build.sh

```diff
@@ -1,73 +1,77 @@
 #!/bin/bash
-#
-#
 
-RELEASE_VERSION=
-buildpath=
+set -e
 
-function setuptree() {
-  b=$( mktemp -d $buildpath/gh-ostXXXXXX ) || return 1
-  mkdir -p $b/gh-ost
-  mkdir -p $b/gh-ost/usr/bin
-  echo $b
-}
-
-function build {
-  osname=$1
-  osshort=$2
-  GOOS=$3
-  GOARCH=$4
+. ./script/common
+buildpath=build
+timestamp=$(date "+%Y%m%d%H%M%S")
 
+function build() {
+  goos=$1
+  osshort=$goos
 
+  envs="-e GOOS=$goos \
+    -e GOARCH=amd64 \
+    -e CGO_ENABLED=1 \
+    -e GOCACHE=/go/src/${PACKAGE_PATH}/.cache \
```

Comment:
This is necessary since we don't run these commands as root inside the container (so we can't write to `/`).

Maintaining these files in the host directory does have a benefit of repeated builds being faster if you keep around the `./.cache` folder.